I pushed your commit, but there is a failing test, at least on my machine:

```
$ python -m unittest tests.unit.modules.timezone_test.TimezoneTestCase.test_get_zone
F
======================================================================
FAIL: test_get_zone (tests.unit.modules.timezone_test.TimezoneTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/unit/modules/timezone_test.py", line 71, in test_get_zone
    self.assertEqual(timezone.get_zone(), '')
AssertionError: '#\nA' != ''
```

--- a/salt/modules/timezone.py
+++ b/salt/modules/timezone.py
@@ -103,7 +103,7 @@ def get_zone():
         ret = __salt__['cmd.run_all'](['timedatectl'], python_shell=False)
 
         if ret['retcode'] > 0:
-            msg = 'timedatectl failed: {}'.format(ret['stderr'])
+            msg = 'timedatectl failed: {0}'.format(ret['stderr'])
             raise CommandExecutionError(msg)
 
         for line in (x.strip() for x in ret['stdout'].splitlines()):

